services/Mailer.js

```diff
@@ -58,7 +58,41 @@ sgClient.setApiKey(keys.sendGridKey);
       console.log('error', e)
     }
   }
+  const sendVerificationMail = async (data) => {
+    const {email} = data;
+
+    try {
+      const existingUser = await User.findOne({ email })
+      if (existingUser) {
+        if (existingUser.googleId) {
+          return res.send('Login with Google')
+        } else {
+          throw Error('User with this email already exists')
+        }
+      }
 
+      const token = Buffer.from(JSON.stringify(data)).toString('base64');
+      const subject = 'Unzipped Verification Link';
+      const msg = {
+        to: email,
+        from: {
+          name: 'Unzipped',
```

Comment:
It would be great if name and email is kept in a constant and then use it here. or in .env